assets/js/components/IngestSheet/Completed.jsx

```diff
@@ -45,17 +51,37 @@ const IngestSheetCompleted = ({ sheetId }) => {
       {ingestSheetErrors.length > 0 && (
         <IngestSheetCompletedErrors errors={ingestSheetErrors} />
       )}
-
-      <div className="columns is-multiline">
-        {works.map((work) => (
-          <div
-            key={work.id}
-            className="column is-half-tablet is-one-quarter-desktop"
-          >
-            <WorkListItem key={work.id} work={work} />
-          </div>
-        ))}
+      <div className="field has-text-right is-hidden-touch">
+        <input
+          id="checkbox-switch-listitem-view"
+          type="checkbox"
+          className="switch"
+          checked={isListView}
+          onChange={(e) => handleWorksViewChange()}
+        />
+        <label htmlFor="checkbox-switch-listitem-view">List View</label>
       </div>
+      {!isListView && (
+        <div className="columns is-multiline">
+          {works.map((work) => (
+            <div
+              key={work.id}
+              className="column is-half-tablet is-one-quarter-desktop"
+            >
+              <WorkCardItem key={work.id} work={work} />
+            </div>
+          ))}
+        </div>
+      )}
+      {isListView && (
+        <div className="">
```

Comment:
If an empty `div`, let's use the React empty div `<> ... </>` to minimize extra markup that gets rendered